src/services/configuration.ts

```diff
@@ -526,7 +525,7 @@ for (const schemaName in schemaContributions.schemas) {
 		if (description) {
 			propertyObject['description'] = description;
 		} else {
-			console.log(`${property}: localize('schema.json.${property}', "")`);
+			console.log(`${property}: l10n.t("")`);
```

Comment:
This looks like the right conversation but I don't think a `l10n` call is needed here since it's an empty string